avalanche-network-runner-sdk/scripts/static-analysis.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+set -xue
+
+if ! [[ "$0" =~ scripts/static-analysis.sh ]]; then
+  echo "must be run from repository root"
+  exit 255
+fi
+
+# git submodule add https://github.com/googleapis/googleapis
+git submodule update --init --remote
+
+# https://rust-lang.github.io/rustup/installation/index.html
```

Comment:
probably some little doc is needed in rust environment installation

Reply:
Added links to https://www.rust-lang.org/tools/install